src/Manager.js

```diff
@@ -19,30 +20,44 @@ class Manager {
     this.forceHashUpdate = debounce(this.handleHashChange, 1)
   }
 
+  setContainer = () => {
+    // if we have a containerId, find the scrolling container, else set it to window
+    if (this.config.containerId) {
+      this.config.container = document.getElementById(this.config.containerId)
```

Comment:
How about `this.config.containerId` could be id's string or container's ref ?
As it is possible that parent component which is using `ScrollableAnchor` may maintain a container's ref~

Reply:
Yeah that's a valid request I guess. I don't think this library is really maintained any more so it's unlikely this will ever get merged

Reply:
I've added a version of this library to my library collection that has some added improvements already (Sadly no documentation yet, i'm working on it though)

https://github.com/leon-marzahn/marzahn-dev/tree/main/libs/react/scroll-anchor